GUI/src/pages/Training/Intents/index.tsx

```diff
@@ -75,6 +77,17 @@ const Intents: FC = () => {
     }
   };
 
+  function queryRefresh(shouldEmptyList: boolean) {
+    if (shouldEmptyList) {
+      intentsFullList = [];
+    }
+    queryClient.cancelQueries(['intents/intents-full']);
+    queryClient.invalidateQueries(['intents/intents-full']);
+    queryClient.fetchQuery(['intents/intents-full']).then(() => {
```

Comment:
Is there a need invalidate and fetch together?

Reply:
I might be wrong, but my thought was that it seemed good for the case that the user clicks to confirm an action in the same moment that the client is already fetching an auto-update. Just in case.